src/components/flow/routers/giftcard/GiftCardRouterForm.tsx

```diff
@@ -66,7 +80,10 @@ class GiftCardRouterForm extends React.PureComponent<RouterFormProps, GiftCardRo
 
   public handleGiftcardChanged = (selected: Asset[], submitting = false): boolean => {
     const updates: Partial<GiftCardRouterFormState> = {
-      giftcardDb: validate('giftcardDb', selected[0], [Required, shouldRequireIf(submitting)])
+      giftcardDb: validate('Giftcard Database', selected[0], [
```

Comment:
Quick question: even we changing the name of the field to `Giftcard Database` they will continue being `giftcardDb` on the flow JSON object, right?

Reply:
Correct, The first parameter is just a name to build validation error. So that's why I have renamed these params.

Reply:
Great!